server/controllers/UsersController.js

```diff
@@ -0,0 +1,241 @@
+import Helpers from '../utils/helper';
+import { User } from '../models';
+
+/**
+ * @class UsersController
+ */
+class UsersController {
+  /**
+   * @description
+   * @param {any} req
+   * @param {any} res
+   * @returns {void}
+   * @memberof UsersController
+   */
+  static registerUser(req, res) {
```

Comment:
I made it this way as the endpoints are just utility functions throughout the application. Do correct me if i'm wrong on this.